mitiq/interface/mitiq_cirq/cirq_utils.py

```diff
@@ -100,3 +104,29 @@ def execute_with_depolarizing_noise(
     rho = simulator.simulate(circuit).final_density_matrix
     expectation = np.real(np.trace(rho @ obs))
     return expectation
+
+
+def generate_inverse_confusion_matrix(
+    qubits: Sequence["cirq.Qid"],
```

Comment:
For this function the specific choice of qubits looks irrelevant for the returned matrix.
Maybe we can simplify this argument with a simpler `num_qubits: int` ?

Reply:
Unfortunately, as this function currently is it does require the actual Cirq `qubits` to be passed due to line 125:
```
qubit_measures = [[q] for q in qubits]
```